supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -71,12 +79,12 @@ class JesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
   private lazy val dockerConfiguration = jesConfiguration.dockerCredentials
 
   override def tryAbort(job: StandardAsyncJob): Unit = {
-    Run(job.jobId, initializationData.genomics).abort()
+    Run(job, initializationData.genomics).abort()
   }
 
   override def requestsAbortAndDiesImmediately: Boolean = true
 
-  override def receive: Receive = pollingActorClientReceive orElse super.receive
+  override def receive: Receive = pollingActorClientReceive orElse runCreationClientReceive orElse super.receive
```

Comment:
tol as they say. I know they're all over the place now but i find these layered receives to be really hard to follow when i'm tracking down message flow